app.js

```diff
@@ -0,0 +1,62 @@
+const express = require("express");
+const Sequelize = require("sequelize");
+var app = express();
+
+const bodyParser = require("body-parser");
+
+const urlencodedParser = bodyParser.urlencoded({extended: false});
+
+const sequelize = new Sequelize("postgres", "postgres", "1234567", {
+  dialect: "postgres"
```

Comment:
Database config should be moved to the configuration file. Use 'config' package for this